### ISSUE TITLE
Basket import of Powershell Framwork checks fails with "Column 'entry_name' cannot be null"

### ISSUE THREAD

**Issue by log1-c** [Icinga/icinga-powershell-framework#83]
With the current version of the PowerShell framework the created Icinga Director basket fails on import with the following error message:

> Storing director_datalist_entry[{"list_id":"11"}] failed: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'entry_name' cannot be null, query was: INSERT INTO director_datalist_entry (list_id, entry_name, entry_value, format, allowed_roles) VALUES (?, ?, ?, ?, ?) {array ( 'list_id' => '11', 'entry_name' => NULL, 'entry_value' => NULL, 'format' => 'string', 'allowed_roles' => NULL, )} (DbObject.php:853)

The reference object seems to be
```
"IcingaCheckCertificate CertStore": {
			"list_name": "IcingaCheckCertificate CertStore",
			"originalId": "2",
			"entries": [
				{
					"allowed_roles": null,
					"entry_value": "*",
					"entry_name": "*",
					"format": "string"
				},
				{
					"allowed_roles": null,
					"entry_value": "LocalMachine",
					"entry_name": "LocalMachine",
					"format": "string"
				},
				{
					"allowed_roles": null,
					"entry_value": "CurrentUser",
					"entry_name": "CurrentUser",
					"format": "string"
				},
				{
					"allowed_roles": null,
					"entry_value": "",
					"entry_name": "",
					"format": "string"
				}
			],
```

At least after adding, e.g. a `"*"` to the emtpy `entry_value `and `entry_name `the import works
```
...
{
					"allowed_roles": null,
					"entry_value": "*",
					"entry_name": "*",
					"format": "string"
				}
... 
```

Not sure if this is a "valid" fix, but at least it works :D

**Comment by log1-c** [comment]
If I see that correctly the deletion of the last part would also be a valid fix.
Is there some CertStore list entry missing or is it just a C&P error?

**Comment by LordHepipud** [comment]
This is already fixed within the Certificate Check-Plugin: [Certificate check fails on director import](https://github.com/Icinga/icinga-powershell-plugins/issues/30).

Because of an invalid default configuration of the Plugin arguments the created Basket is causing a conflict. Once the Plugin is updated the issue should be fixed.

**Comment by log1-c** [comment]
@LordHepipud Is this already part of the 1.1.0 release of the plugins, if I install them today, or do I have to use a different version or the snapshot tag?

**Comment by LordHepipud** [comment]
Hello,

sorry for not mentioning this! The fix is in the current master branch only. Installing the snapshot will resolve the issue.
The fix is part of 1.2.0 which will be released somewhere in August.
